src/cluster_legacy.c

```diff
@@ -6096,130 +6575,7 @@ int clusterCommandSpecial(client *c) {
         /* SETSLOT 10 IMPORTING <node ID> */
         /* SETSLOT 10 STABLE */
         /* SETSLOT 10 NODE <node ID> */
-        int slot;
-        clusterNode *n;
-
-        if (nodeIsSlave(myself)) {
-            addReplyError(c,"Please use SETSLOT only with masters.");
```

Comment:
Worth noting that replicas of older versions will get here when replicating from a Valkey 8 node. IIRC, this error logged on the replica, but it may be worth checking and maybe mention in the PR description and/or docs.

Reply:
At some point in the future we want the replica to crash when it encounters an error from it's primary, or at least I wrote https://github.com/valkey-io/valkey/blob/8abeb79f52cbf02d16d546f48e16592173593132/valkey.conf#L757 at one point.

Reply:
So this will actually crash replicas with this config set to panic?

In that case, I think we should use a cluster node flag as a capability flag to indicate support for this feature. If the replica doesn't have this flag set, we don't replicate SETSLOT to it.

It'd be similar to the flag `CLUSTER_NODE_EXTENSIONS_SUPPORTED`.

Reply:
I opened follow-up issue #421 for this. Now we can merge without it. :)